### PR TITLE
UAF-84 UAF-3353 Attributes on Award. New BO created, maintenance doc …

### PR DESCRIPTION
**Pull request by mcalopez**
…and datadictionary for Award were modified.

### CODE REVIEW

File: kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/AwardExtension.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			<list>
+				<ref bean="AwardExtension-proposalNumber" />
+				<ref bean="AwardExtension-federalPassThroughPrimeAwardNumber" />
+			</list>
+		</property>
+	</bean>
+
+	<!-- Attribute Definitions -->
+
+	<bean id="AwardExtension-proposalNumber" parent="AwardExtension-proposalNumber-parentBean" />
```
Comment (ghost) [inline kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/AwardExtension.xml]: We can replace these two beans with the simpler:
`<bean id="AwardExtension-proposalNumber" parent="Award-proposalNumber-parentBean" />` 
This property is the primary key of the BO being extended, so we can inherit from there.

File: kfs-core/src/main/resources/institutional-config.properties [hunk 2]
```diff
@@ -59,11 +59,11 @@ spring.source.files=classpath:spring-kfs-imported-rice-beans.xml, \
   classpath:edu/arizona/kfs/fp/spring-fp.xml, \
```
Comment (ghost) [inline kfs-core/src/main/resources/institutional-config.properties:59]: I don't see why this change was needed as an edu.arizona version of spring-cg.xml was already included in the list of spring source files.

**Review by ghost — COMMENTED** [review]
Initial review completed. Please see my comments.

File: kfs-core/src/main/resources/institutional-config.properties (hunk 2 above)
Comment (mcalopez) [reply kfs-core/src/main/resources/institutional-config.properties:59]: For some reason, leaving classpath:edu/arizona/kfs/module/cg/spring-cg.xml at line 62 would make the spring file not able to read the cgModuleConfiguration bean, it won't throw an error but the new Award.xml and AwardExtension.xml were not being read; even when I purposely misspelled the parent="cgModuleConfigurateion-parentBean", the spring-cg would not throw an error. 
However, when moving this line at the bottom of the list then everything will work as expected. It would throw an exception when purposely misspelling the parent bean on the spring file.

Comment (ghost) [reply kfs-core/src/main/resources/institutional-config.properties:59]: OK

**Comment by ghost** [comment]
Looks good!
